src/it/scala/net/snowflake/spark/snowflake/SnowflakeResultSetRDDSuite.scala

```diff
@@ -710,111 +710,102 @@ class SnowflakeResultSetRDDSuite extends IntegrationSuiteBase {
 
   test("testTimestamp") {
     setupTimestampTable
-    // COPY UNLOAD can't be run because it only supports millisecond(0.001s).
-    if (!params.useCopyUnload) {
-      val result = sparkSession.sql("select * from test_table_timestamp")
+    val result = sparkSession.sql("select * from test_table_timestamp")
```

Comment:
This test case failed.

Reply:
@Mingli-Rui Yea sorry, haven't finished the change yet, should've mentioned this. Will push another commit this week.